ext/fg/js/popup-factory.js

```diff
@@ -21,7 +21,7 @@
  * apiFrameInformationGet
  */
 
-class PopupProxyHost {
+class PopupFactory {
```

Comment:
Updated in a1b8a8c6018f5366f2e91bcaf365854545a12670, 840284350a058a1a40718a5d33d8723e2a41d233, 13bbfcd0ff850a5fef33502672f7f29b8df56774. I think at some point it may be good to bring the `getUrl` function out of `PopupProxy`, since it's actually used by `Frontend`. This would likely also involve passing the `FrontendApiSender` instance to `PopupProxy`'s constructor, so this can all be done in a later commit.

Reply:
> Updated in [a1b8a8c](https://github.com/FooSoft/yomichan/commit/a1b8a8c6018f5366f2e91bcaf365854545a12670), [8402843](https://github.com/FooSoft/yomichan/commit/840284350a058a1a40718a5d33d8723e2a41d233), [13bbfcd](https://github.com/FooSoft/yomichan/commit/13bbfcd0ff850a5fef33502672f7f29b8df56774)

Looks good :+1: 

> I think at some point it may be good to bring the `getUrl` function out of `PopupProxy`, since it's actually used by `Frontend`

Yeah, I was also trying to find other options but using the proxy host was the simplest I found at the time.